module/documents/actor/actor.mjs

```diff
@@ -365,7 +365,8 @@ export default class Actor5e extends Actor {
 
       abl.save = abl.mod + abl.saveBonus;
       if ( Number.isNumeric(abl.saveProf.term) ) abl.save += abl.saveProf.flat;
-      abl.dc = 8 + abl.mod + this.system.attributes.prof + dcBonus;
+      abl.spellDc = 8 + abl.mod + this.system.attributes.prof + spellDcBonus;
```

Comment:
Probably breaking change: renamed this to better represent what it does
Thoughts?

Reply:
While I'll leave it to the project maintainers to approve/reject this, keep in mind that a change like this requires:
  - [Migrating}(https://github.com/foundryvtt/dnd5e/blob/f2c7c99193f5c00d327e9fb515b3759919b86e91/module/migration.mjs) existing actor data.
  - Changing the data model in [template.json](https://github.com/foundryvtt/dnd5e/blob/master/template.json).
  - Renaming any other references to this attribute.